src/commands/OwnersCommand.js

```diff
@@ -0,0 +1,47 @@
+import Command from "../Command";
+import NpmUtilities from "../NpmUtilities";
+import ConfigUtilities from "../ConfigUtilities";
+import async from "async";
+
+export default class OwnersCommand extends Command {
+  initialize(callback) {
+    this.command = this.input[0];
+    this.ownerName = this.input[1];
+
+    if (!this.command || !["rm", "add"].includes(this.command)) {
+      callback(new Error("You must specify which command to run, and it must be one of: add, rm"));
+      return;
+    }
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    let config = ConfigUtilities.read(this.repository.rootPath);
+    if (!config.owners) {
+      config.owners = [];
+    }
+    const ownersSet = new Set(config.owners);
+    if (this.command === "add") {
+      ownersSet.add(this.ownerName);
+    } else if (this.command === "rm") {
+      ownersSet.delete(this.ownerName);
+    }
+    config.owners = [...ownersSet];
+    ConfigUtilities.write(this.repository.rootPath, config);
+    async.parallelLimit(this.filteredPackages.map((pkg) => (cb) => {
+      this.runCommandInPackage(pkg, cb);
+    }), this.concurrency, callback);
+  }
+
+  runCommandInPackage(pkg) {
+    const pkgName = pkg._package.name;
```

Comment:
There's actually [a `name` getter](https://github.com/lerna/lerna/blob/master/src/Package.js#L13-L15), so you should be able to just do `pkg.name` here.